srtp_cipher_aes_cm_hmac_sha1.go

```diff
@@ -23,34 +23,36 @@ type srtpCipherAesCmHmacSha1 struct {
 	srtcpBlock       cipher.Block
 }
 
+const srtpCipherAesCmHmacSha1AuthKeySize = 20
```

Comment:
I think this is better to be `ProtectionProfile.authKeyLen()` instead of const.

Reply:
AES-GCM doesn't have an authKeyLen, ok if I add `authKeyLen` to `srtpCipherAesCmHmacSha1` ?

Reply:
For AES-GCM case, returning 0 would be just fine.

Reply:
Done!